IFC4x3/Properties/s/SpanLength_0oTEbvO096Gv87GKVYXQSy/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="SpanLength_0oTEbvO096Gv87GKVYXQSy" Name="SpanLength" UniqueId="3274e979-6002-4643-9207-4147e285a73c" PrimaryDataType="IfcPositiveLengthMeasure" />
```

Comment:
Can't find this property in our PSET

Reply:
I don't get this point. we are here using Span_0PY8o0qSGHuO00025QrEV 
Should we not use the first concept listed? The other one is Span_1Ttb60qV4HuO00025QrE$V

Reply:
Might have been changed in one of the follow-up commits but this initial review has SpanLength_0oTEbvO096Gv87GKVYXQSy. When was it changed so that I can resolve?